src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/FinremCaseData.java

```diff
@@ -667,6 +694,26 @@ public boolean isRespondentSolicitorPopulated() {
         return StringUtils.isNotEmpty(nullToEmpty(getContactDetailsWrapper().getRespondentSolicitorEmail()));
     }
 
+    @JsonIgnore
+    public boolean isIntervenerSolOnePopulated() {
+        return StringUtils.isNotEmpty(nullToEmpty(getIntervenerOneWrapper().getIntervener1SolEmail()));
+    }
+
+    @JsonIgnore
+    public boolean isIntervenerSolTwoPopulated() {
+        return StringUtils.isNotEmpty(nullToEmpty(getIntervenerTwoWrapper().getIntervener2SolEmail()));
```

Comment:
instead of these two method, better to use one Objects.toString( arg1, default)

Reply:
Went with this method based on advice/discussion with Ruban during implementation